menu.py

```diff
@@ -47,6 +49,35 @@ def menu():
             case "5":
                 add_new_product()
 
+            case "6":
+                sort_option = input("""
+            Wybierz sposób sortowania:
+            [a] Cena rosnąco
+            [b] Cena malejąco
+            [c] Producent A-Z
+            [d] Producent Z-A
+            [e] Nazwa A-Z
+            [f] Nazwa Z-A
+            [g] Czas dostawy (malejąco)
```

Comment:
I'd change the naming of the options so it's  more intuitive to the user. E.g.  "Cena rosnąco" --> "Od największej ceny" , not necessary tho. What do you think?

Reply:
I'd leave it as it is. It's shorter and faster to read.